Orange/widgets/visualize/owfreeviz.py

```diff
@@ -372,7 +372,8 @@ def get_embedding(self):
             return None
         EX = np.dot(self._X, self.projection)
         EX /= np.max(np.linalg.norm(EX, axis=1)) or 1
-        embedding = np.zeros((len(self.data), 2), dtype=np.float)
+        embedding = np.empty((len(self.data), 2), dtype=np.float)
+        embedding.fill(np.nan)
```

Comment:
I guess `np.full((len(self.data), 2), np.nan)` does the same job as these two lines. Or is there a difference?

No need to change this PR, I'm just saying it for next time.

Reply:
Have changed it anyway :)